packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/TinyMCE.tsx

```diff
@@ -396,7 +396,7 @@ const TinyMCE = (props: NoteBodyEditorProps, ref: any) => {
 		document.head.appendChild(element);
 		element.appendChild(document.createTextNode(`
 			.joplin-tinymce .tox-editor-header {
-				padding-left: ${styles.leftExtraToolbarContainer.width + styles.leftExtraToolbarContainer.padding * 2}px;
+				padding-left: ${document.getElementById('leftExtraToolbarContainer').offsetWidth}px;
```

Comment:
I mean the component state as set by `useState`. The pattern I think would be to listen for size change events on the left toolbar, then save the width to a state variable. Then that state variable should be used as a dependency to this effect, and then used here.

Reply:
Thanks, now got it...